tests/foreman/cli/test_host.py

```diff
@@ -2041,6 +2043,32 @@ def test_syspurpose_end_to_end(
         )
 
 
+@pytest.mark.e2e
+def test_positive_register_read_bootc(target_sat, bootc_host, function_ak_with_cv, function_org):
+    """Register a bootc host and validate the information
+
+    :id: d9557843-4cc7-4e70-a035-7b2c4008dd5e
+
+    :expectedresults: Upon registering a Bootc host, the facts are attached to the host, and are accurate.
+
+    :CaseComponent:Hosts-Content
```

Comment:
Please confirm the component. I think this test should be part of Hosts component, as it just a registering host and validating its information and nothing related to content manipulation.
```suggestion
    :CaseComponent:Hosts
```